pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -104,6 +104,30 @@ type BMCDetails struct {
 	CredentialsName string `json:"credentialsName"`
 }
 
+// BootMode is the boot mode of the system
+type BootMode string
+
+// Allowed boot mode from metal3
+const (
+	Uefi BootMode = "uefi"
```

Comment:
Could this be named `UEFI` instead (all caps), as it's an acronym?

Reply:
Thanks @stbenjam . I will update this name to `UEFI` :)